### PR TITLE
Add basic i18n support

### PR DESCRIPTION
**Pull request by NKjoep**
Hi 👋 

I saw the request for basic support and came up with this. It might help someone. 

I've added a new test suite with the Italian localization, the tests are still valid. Have a look.

### CODE REVIEW

**Review by lukeed — CHANGES_REQUESTED** [review]
Hey, thanks!

Few things:

1) ~~Can you please revert the filename change for tests?~~ _My bad_

2) Let's drop the `periods` object and make it flat/single-level object.

3) No need for `now` key; it's not being used anywhere.

4) Let's take all this `i18n` logic and move it into a new entry file: `src/i18n.js`. I will then follow up this PR with `builder.js` changes to expose it correctly. The idea here is that most people may not care/want the i18n support, but those who do can do `import fromNow from 'fromnow/i18n`.

Thanks!

File: README.md [hunk 1]
```diff
@@ -111,6 +111,32 @@ fromNow("Sun Jun 14 2015 15:14:05", { zero:true });
...
+Type: `Object`<br>
+Default: `undefined`
+
+Set the localisation.
+
+```js
+var i18nData = {
+  ago: 'fa',
+  now: 'adesso',
+  fromnow: 'da ora',
```
Comment (lukeed) [inline README.md]: Let's rename this key `future`.

File: README.md [hunk 2]
```diff
@@ -111,6 +111,32 @@ fromNow("Sun Jun 14 2015 15:14:05", { zero:true });
...
 
+#### options.i18n
+Type: `Object`<br>
+Default: `undefined`
+
+Set the localisation.
+
+```js
+var i18nData = {
+  ago: 'fa',
```
Comment (lukeed) [inline README.md]: And rename this one `past`

File: README.md [hunk 3]
```diff
@@ -111,6 +111,32 @@ fromNow("Sun Jun 14 2015 15:14:05", { zero:true });
...
+  now: 'adesso',
+  fromnow: 'da ora',
+  justnow: 'proprio ora',
```
Comment (lukeed) [inline README.md]: Finally, `justnow` can become `now`

File: src/index.js [hunk 4]
```diff
@@ -4,13 +4,30 @@ var DAY = HOUR * 24;
...
+		month: ['month', 'months'],
+		day: ['day', 'days'],
+		hour: ['hour', 'hours'],
+		minute: ['minute', 'minutes'],
+	}
+}
+
 export default function (date, opts) {
 	opts = opts || {};
+	var translations = Object.assign({}, I18N, opts['i18n']);
```
Comment (lukeed) [inline src/index.js]: No need for `Object.assign` here

```js
var translations = opts.i18n || I18N;
// as per review request, "periods" is gone anyway
```

**Comment by NKjoep** [comment]
Hey Luke, thanks for reviewing it! I did _2)_ and _3)_ (renamed the keys, flattened etc).

I am not sure I understand how you would like to proceed with _4)_ and i18n abstraction. 🙄
The only logic added is the `var translations = opts.i18n || I18n`.

What I would do is to, as you said, move away the `i18n` constant (or constants) so that it can be skipped in the final build.

Please give me some more instruction so I can complete this, or feel free to close PR and proceed independently. No offence taken :D

**Pull request closed without merging**